src/lib/stores/workflows.ts

```diff
@@ -57,6 +58,15 @@ export const createStore = (namespace: string) => {
   const ids = derived(store, ($store) => Object.keys($store.ids));
   const range = createStoreWithCallback<Duration>({ hours: 24 }, update);
 
+  range.subscribe(() =>
+    store.set({
+      loading: false,
+      updating: true,
+      workflows: {},
```

Comment:
will this make the workflows screen render empty before fetching the data again? 
If so, what happens if we filter the existing store workflows based on range and fetch additional workflows in parallel?

also not a concern for this PR, just a side note

Reply:
It will, for now. But, that's only a stop-gap measure. I had a bug where it wasn't removing stale executions. A forthcoming refactor will get this right.